.github/workflows/ci.yml

```diff
@@ -18,3 +18,26 @@ jobs:
       - name: Generate protobuf code
         run: protoc --js_out=/tmp embedded_sass.proto
 
+  tag:
+    name: "Tag release"
+    runs-on: "ubuntu-latest"
+    if: "github.event_name == 'push' && github.repository == 'sass/embedded-protocol'"
```

Comment:
The expression syntax is `${{ expression }}` instead of `"expression"`

Additionally, this should be restricted to `github.ref == 'refs/heads/main'`.